0800-appendix-198215-dale.Rmd

```diff
@@ -261,3 +297,52 @@ knitr::include_graphics(my_photo1)
 knitr::include_graphics("fig/pixel.png")
 knitr::include_graphics(my_photo2)
 ```
+
+```{r photo-198215-07-prep}
+my_photo1 = fpr::fpr_photo_pull_by_str(site = my_site, dir_photos = "data/photos/", str_to_pull = '_ct_us')
+
+my_caption1 = paste0('Cutthroat trout captured upstream of PSCIS crossing ', my_site, '.')
+
+```
+
+```{r photo-198215-07, fig.cap= my_caption1, out.width = photo_width, eval=gitbook_on}
+grid::grid.raster(jpeg::readJPEG(my_photo1))
```

Comment:
fyi - pretty sure `knitr::include_graphics` can replace `grid::grid.raster(jpeg::readJPEG())`.  There is an issue in one of the 2022 repos about a bug that was overcome where we needed `grid::grid.raster(jpeg::readJPEG())` as a workaround.  I don't think we need to replace it everywhere but going forward we can use `knitr::include_graphics` because it uses 1 package instead of 2, is less code and presents a better looking figure (white lines around the photo in grid:: version)

You can test and confirm for yourself.  If I'm wrong please let me know